### PR TITLE
net/frr: add some OSPFv3 features and fixes

### LINKED ISSUE #1418: net/frr: add passive Interface to ospfv3
https://twitter.com/STRUBartacus/status/1151916658707050498?s=03

### PR DESCRIPTION
**Pull request by mimugmail**
Closes #1418 

* add changelog for 1.12 to respect other commits
* fix network type for ospfv3
* add passive-interface for ospfv3

### CODE REVIEW

File: net/frr/src/opnsense/mvc/app/models/OPNsense/Quagga/OSPF6.xml [hunk 1]
```diff
@@ -42,6 +42,10 @@
                                 <Required>Y</Required>
                                 <mask>/^\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}$/</mask>
                         </area>
+                        <passive type="BooleanField">
+                                <default>0</default>
+                                <Required>N</Required>
```
Comment (fabianfrz) [inline net/frr/src/opnsense/mvc/app/models/OPNsense/Quagga/OSPF6.xml]: should be required - will be defaulted anyway especially when you expect it to be there.

File: net/frr/src/opnsense/mvc/app/models/OPNsense/Quagga/OSPF6.xml [hunk 2]
```diff
@@ -42,6 +42,10 @@
                                 <Required>Y</Required>
```
Comment (fabianfrz) [inline net/frr/src/opnsense/mvc/app/models/OPNsense/Quagga/OSPF6.xml]: version bump the model

File: net/frr/src/opnsense/mvc/app/models/OPNsense/Quagga/OSPF6.xml (hunk 1 above)
Comment (fabianfrz) [reply net/frr/src/opnsense/mvc/app/models/OPNsense/Quagga/OSPF6.xml]: Did you change the wrong property?

Comment (mimugmail) [reply net/frr/src/opnsense/mvc/app/models/OPNsense/Quagga/OSPF6.xml]: Whoopsie .. sorry

**Comment by fichtner** [comment]
@fabianfrz all good now?

**Comment by fabianfrz** [comment]
@fichtner yes

**Comment by fichtner** [comment]
@fabianfrz please please please don't merge if you're not the assignee.

**Comment by fabianfrz** [comment]
OK, just thought you already want to merge because of the way you asked the previous question?

**Comment by fichtner** [comment]
no, you didn't approve until I asked because I wasn't going to merge without your ok--the status was still "changes requested"--following a bit of procedure doesn't hurt otherwise review and assignee are irrelevant and the whole thing is a mess for the author of the PR.

**Comment by fabianfrz** [comment]
Ok

**Pull request merged**